content/concept-xrp-ledger-intro.md

```diff
@@ -0,0 +1,88 @@
+
+## Censorship-Resistant Transaction Processing
+[Censorship-resistant transaction processing]: #censorship-resistant-transaction-processing
+
+With (physical) coins and paper money, individuals can do business without going through a central party. Non-cryptocurrency digital assets all have a central administrator to confirm and validate transactions, who also has the power to censor or roll back transactions, or disallow some individuals from using the digital asset. (For example, if a digital money company decides a person has violated its terms of service, it can freeze or even confiscate that person's money.) XRP and other cryptocurrencies don't have a central administrator, so no one can roll back transactions, freeze XRP balances, or block someone from using them.
+
+**Note:** Non-XRP currencies issued in the XRP Ledger _can_ be frozen. For more information, see the [Freeze documentation](concept-freeze.html).
+
+Furthermore, the XRP Ledger's system of trusted validators uses a small amount of human interaction to achieve better distribution of authority than other decentralized systems. Fully-automated systems for reaching consensus from an unknown set of participants are vulnerable to concentrations of voting power. For example, Bitcoin mining is disproportionately concentrated in places with cheap electricity. As Ripple curates a list of distinct validators operated by different entities in different jurisdictions, the XRP Ledger can become more resistant to censorship and outside pressures than proof-of-work mining. For more information on Ripple's plan to decentralize the recommended set of validators, see the  [Decentralization Strategy Update](https://ripple.com/dev-blog/decentralization-strategy-update/).
+
+
+## Fast, Efficient Consensus Algorithm
+[Fast, Efficient Consensus Algorithm]: #fast-efficient-consensus-algorithm
+
+The XRP Ledger's biggest difference from most cryptocurrencies is that it uses a unique consensus algorithm that does not waste time and energy on "mining", the way Bitcoin, Etherium, and almost all other such systems do. Instead of "proof of work" or even "proof of stake", The XRP Ledger's consensus algorithm uses a system where every participant has an overlapping set of "trusted validators" and those trusted validators efficiently agree on which transactions happen in what order. A single Bitcoin transaction wastes more electricity than a family home in the USA uses in an entire day, and confirming the transaction takes hours. A single XRP transaction uses a negligible amount of electricity, and takes 4 or 5 seconds to confirm. Furthermore, each new "ledger version" in the XRP Ledger (the equivalent of a "block") contains the full current state of all balances, so a server can synchronize with the network in minutes instead of spending hours downloading and re-processing the full transaction history.
```

Comment:
Change "Etherium" to "Ethereum"

Change "of stake", The XRP Ledger's" to "of stake", the XRP Ledger's"

Consider changing "wastes more electricity" to "uses more electricity"

Reply:
"A single Bitcoin transaction wastes more electricity than a family home in the USA uses in an entire day" - this is misleading and/or wrong, the same goes for the second part of that sentence. It is also irrelevant to an explanation about how XRPL works.

Reply:
Respectfully, the fact that the XRP Ledger does not incentivize a competition for who can muster the most computing power is a _very_ relevant part of how the XRP Ledger works. It motivates core design decisions, is the _biggest_ differentiator from proof-of-work cryptocurrencies, and is potentially very important for the future of the planet.

I've rephrased the first clause to be less incendiary, though:

> As of early 2018, the amount of electricity the Bitcoin network uses per transaction is more than a family home in the USA uses in an entire day, and confirming the transaction takes hours.

Reply:
"estimated to use" - and the usage is per block, not per transaction. Confirmation also doesn't take hours, it doesn't even take a single hour on average (even for 6 blocks, which is already something I would only require for _very_ high value transactions)